packages/api/src/schema/api-schema.yaml

```diff
@@ -8,8 +8,71 @@ info:
   version: 1.0.0
 servers:
   - url: "https://livepeer.studio/api"
+tags:
+  - name: Asset
+  - name: Stream
+  - name: Webhook
+  - name: Room
+  - name: Signing Key
+  - name: Multistream
+  - name: Session
+  - name: Task
+  - name: Viewership
+  - name: Misc
 components:
   schemas:
+    request-upload:
+      type: object
+      required:
+        - name
+      properties:
+        name:
+          type: string
+          example: Example name
```

Comment:
Started using `new-asset-payload` on the API schemas, but kept the `url` field tech debt for now. It will be shown for the `/request-upload` API even though it is not used there. To be fixed later.